test-data/unit/lib-stub/typing.pyi

```diff
@@ -78,7 +78,11 @@ class Sequence(Iterable[T], Generic[T]):
     @abstractmethod
     def __getitem__(self, n: Any) -> T: pass
 
-class Mapping(Generic[T, U]): pass
+class Mapping(Generic[T, U]):
+    @overload
+    def get(self, k: T) -> Optional[U]: pass
```

Comment:
Can you move this to `Dict` in `fixture/dict.pyi`? We try to keep the `typing.pyi` stub pretty minimal.